### PR TITLE
[15.0][IMP] dms_field: Add #id from record to auto-generate access group to prevent error

### PR DESCRIPTION
**Pull request by victoralmau**
Add #id from record to auto-generate access group to prevent error

Example use case:
- Create an employee in company A (auto-generated group is created).
- Archive employee
- Create the same employee in company B (auto-generated group is created).

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT49176

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]

File: dms_field/i18n/dms_field.pot [hunk 1]
```diff
@@ -164,7 +166,6 @@ msgstr ""
 #: model:ir.model.fields,field_description:dms_field.field_decimal_precision__dms_directory_ids
 #: model:ir.model.fields,field_description:dms_field.field_dms_access_group__dms_directory_ids
 #: model:ir.model.fields,field_description:dms_field.field_dms_category__dms_directory_ids
-#: model:ir.model.fields,field_description:dms_field.field_dms_classification_template__dms_directory_ids
```
Comment (pedrobaeza) [inline dms_field/i18n/dms_field.pot]: Please remove all these changes from the files, as you locally don't have all the modules installed. Anyway, this makes me wonder if we should have this field populated for all the models:

https://github.com/OCA/dms/blob/d926f50e5e41a848d591cd7b28382da9cef040a1/dms_field/models/base.py#L11

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

File: dms_field/i18n/dms_field.pot (hunk 1 above)
Comment (victoralmau) [reply dms_field/i18n/dms_field.pot]: Changes removed. IMO I think that `dms_directory_ids` field should exist in `dms`, but I think it's something that can be discussed (along with other `dms_field` stuff) in the 16.0 migration.

Comment (pedrobaeza) [reply dms_field/i18n/dms_field.pot]: OK, I ping @etobella for his knowledge.

**Review by etobella — APPROVED** [review]
LGTM Makes complete sense

**Comment by pedrobaeza** [comment]
Well, @etobella my ping was about the off-topic opened here about having `dms_directory_ids` field populated in every model, causing that ugly effect in the translations, instead of having a mixin.

**Pull request merged**